Calibration/TkAlCaRecoProducers/python/ALCARECOPromptCalibProdSiPixelLorentzAngleMCS_cff.py

```diff
@@ -65,5 +66,6 @@
     ALCARECOCalCosmicsFilterForSiPixelLorentzAngleMCS *
     ALCARECOPixelLATrackFilterRefitMCS *
     ALCARECOSiPixelLACalibMCS *
-    MEtoEDMConvertSiPixelLorentzAngleMCS 
+    MEtoEDMConvertSiPixelLorentzAngleMCS,
+    cms.Task(SiPixelTemplateStoreESProducer)
```

Comment:
for my education, why is this line with `cms.Task()` while none of the above are? Thanks!

Reply:
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideAboutPythonConfigFile#Behavior_when_an_ESProducer_ESSo

Reply:
I see, thanks Marco and Chris!